jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarProcessor.java

```diff
@@ -49,4 +66,88 @@ public static JarType determineJarType(Path jarPath) throws IOException {
     }
     return JarType.STANDARD;
   }
+
+  /**
+   * Explode jar and create three layers for classes, resources and dependencies on container.
+   *
+   * @param jarPath path to jar file.
+   * @return list of {@link FileEntriesLayer}.
+   * @throws IOException if I/O error occurs when opening the file.
+   */
+  public static List<FileEntriesLayer> explodeStandardJar(Path jarPath) throws IOException {
+    TempDirectoryProvider tempDirectoryProvider = new TempDirectoryProvider();
```

Comment:
I think this should be configurable and if not then use a temporary directory.

Potentially we could have a jib local build directory that we can pass in.

Reply:
Thanks for pointing this out.